.github/workflows/reusable-federation-tests.yml

```diff
@@ -16,6 +16,8 @@ jobs:
           cache: yarn
       - name: Install dependencies
         run: yarn
+      - name: Install webpack-cli
+        run: yarn add -D webpack-cli
```

Comment:
doesn't this also add webpack to the package json?

not important, as it is going to be removed anyway, but maybe a simple `npm i webpack-cli` (or the equivalent yarn) is a bit more accurate to what we try to do here